cli/operator/node.go

```diff
@@ -156,8 +167,13 @@ var StartNodeCmd = &cobra.Command{
 			go startMetricsHandler(cmd.Context(), logger, db, cfg.MetricsAPIPort, cfg.EnableProfile)
 		}
 
-		metrics.WaitUntilHealthy(logger, cfg.SSVOptions.Eth1Client, "execution client")
-		metrics.WaitUntilHealthy(logger, cfg.SSVOptions.BeaconNode, "consensus client")
+		nodeChecker.Wait()
+
+		nodeUnreadyHandler := func() {
+			logger.Fatal("Ethereum nodes are out of sync. Usage of unsynchronized nodes can lead to undefined behavior, such as wrongly marking removed validators as active. Ensure the nodes are synced to resume.")
```

Comment:
@nkryuchkov I think this can be a comment.
In the log we should state which node is out of sync (execution layer or consensus layer)
so it will be more informative.
so something like the execution node is out of sync -